### DISCUSSION TITLE
(solved) Aligning badges horizontally in restructured text pages

### DISCUSSION THREAD

**Discussion by mccroweyclinton-EPA** [community/community#67857] [Repositories] [answered]
### Select Topic Area

Question

### Body

For some reason I can not get my sheilds.io badges on my README to align horizontally in my Restrructured Text README file. Can someone please tell me how to fix that?

[https://github.com/USEPA/pyaqsapi#readme](https://github.com/USEPA/pyaqsapi#readme)

**Comment by seyLu** [comment]
#### Hi @mccroweyclinton-EPA

Alignment issues in images often happen due to difference in image height:
<table>
<td>
<img src="https://github.com/community/community/assets/98249191/a7eb2537-e770-4864-aeef-a6ec8b78bd39">
</td>
<td>
<img src="https://github.com/community/community/assets/98249191/2596db5a-b7a4-4249-a877-5e3ca2ac3b22">
</td>
</table>

Currently, there is no way to specify height of badge directly on `shields.io` endpoint.
What you can do though:
- choose a badge style and stick with it. Different badge styles have different heights as in this case.
![image](https://github.com/community/community/assets/98249191/58c7b6ed-1773-4432-a9ef-fb94e60f49cd)
- I am unsure if you can use html with rst, but if you can, then it is possible to specify height:
```html
<img height="<img_height>" src="...">
```

You can read more on:
- [Shields.io endpoint docs](https://shields.io/badges/endpoint-badge)

**Comment by mccroweyclinton-EPA** [comment, marked as answer]
I am sorry, I should have updated this post. I made a commit that fixed the original issue. I am working on the height alignment now.

Thank you.

**Comment by mccroweyclinton-EPA** [comment]
it is now fixed
